src/bundle.c

```diff
@@ -2041,7 +2041,10 @@ gboolean check_bundle(const gchar *bundlename, RaucBundle **bundle, CheckBundleP
 		}
 		g_debug("Downloaded temp bundle to %s", ibundle->path);
 #else
-		g_warning("Mounting remote bundle not supported, recompile with -Dnetwork=true");
+		g_set_error(error, R_BUNDLE_ERROR, R_BUNDLE_ERROR_FORMAT,
```

Comment:
Not sure how well `R_BUNDLE_ERROR_FORMAT` fits here which is used otherwise to show an issue with the bundle itself. Maybe define a new generic one like `R_BUNDLE_ERROR_UNSUPPORTED`.